### PR TITLE
142 search

### LINKED ISSUE #169: update schema for blood type biospecimen.
Add anatomic site and tissue type for blood sample

### PR DESCRIPTION
**Pull request by mingjiecn**
close #169 , #168 , #152 , #146 , #142

### CODE REVIEW

File: src/encoded/schemas/biospecimen.json [hunk 1]
```diff
@@ -201,6 +206,18 @@
...
+        },
+        "surgery.pathology_report.n_stage": {
+            "title": "pN stage"
+        },
+        "surgery.pathology_report.m_stage": {
+            "title": "pM stage"
+        },
+        "surgery.pathology_report.ajcc_tnm_stage": {
+            "title": "TNM stage"
         }
```
Comment (SenseCode) [inline src/encoded/schemas/biospecimen.json:221]: Upper case with title

File: src/encoded/static/components/patient.js [hunk 2]
```diff
@@ -53,6 +54,33 @@ class Patient extends React.Component {
...
+      }
+      if (i != surgeryData.length - 1) {
+        list.push(<div className="row" style={{ borderTop: "1px solid #151313" }}></div>)
+      }
+
+
+    }
+    return list
+
+  }
```
Comment (SenseCode) [inline src/encoded/static/components/patient.js:83]: redundant codes?

File: src/encoded/static/components/patient.js [hunk 3]
```diff
@@ -109,7 +141,9 @@ class Patient extends React.Component {
     const surgeryPanelBody = (
       <SurgeryChart chartId="surgery" data={context.surgery} chartTitle="Surgeries Results Over Time"></SurgeryChart>
     );
-
+    const pathPanelBody = (
```
Comment (SenseCode) [inline src/encoded/static/components/patient.js:144]: redundant codes?

File: src/encoded/static/components/patient.js [hunk 4]
```diff
@@ -109,7 +141,9 @@ class Patient extends React.Component {
...
+      <dl className="key-value">{this.createPathPanel()}</dl>
+    );
```
Comment (SenseCode) [inline src/encoded/static/components/patient.js:146]: Redundant codes? Didn't show used of this.createPathPanel()

File: src/encoded/static/components/patientPathTable.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+    componentDidMount() {
+        
+        this.renderData();
+    }
+
+}
+
+export default patientPathTable;
+
```
Comment (SenseCode) [inline src/encoded/static/components/patientPathTable.js:106]: image
Seems a bug there with repeated PR.

File: src/encoded/schemas/biospecimen.json (hunk 1 above)
Comment (mingjiecn) [reply src/encoded/schemas/biospecimen.json:221]: Good catch. Will fix. Thanks!

File: src/encoded/static/components/patient.js (hunk 2 above)
Comment (mingjiecn) [reply src/encoded/static/components/patient.js:83]: Will remove. Thanks!

File: src/encoded/static/components/patient.js (hunk 4 above)
Comment (mingjiecn) [reply src/encoded/static/components/patient.js:146]: Will remove. Thanks!

File: src/encoded/static/components/patientPathTable.js (hunk 5 above)
Comment (mingjiecn) [reply src/encoded/static/components/patientPathTable.js:106]: Fixed. Thanks!

**Review by vsmalladi — APPROVED** [review]

**Pull request merged**